plugins/BEdita/Core/src/Service/Mail.php

```diff
@@ -14,21 +14,113 @@
 namespace BEdita\Core\Service;
 
 use BEdita\Core\Job\JobService;
+use Cake\Core\Configure;
+use Cake\Mailer\Email;
+use Cake\Mailer\Exception\MissingMailerException;
+use Cake\Mailer\MailerAwareTrait;
+use Cake\Utility\Hash;
 
 /**
- * Send single mail
- * [Temporary dummy implementation]
+ * Service to send single mail
  *
  * @since 4.0.0
  */
 class Mail implements JobService
 {
+    use MailerAwareTrait;
 
     /**
-     * {@inheritDoc}
+     * Email class
+     *
+     * @var \Cake\Mailer\Email
+     */
+    protected $Email;
+
+    /**
+     * Send a single email directly or using a mailer class.
+     *
+     * $payload array data specify mail action and input data.
+     * Possible keys are:
+     *  * 'profile' - Email configuration profile to use (default is 'default'), in profile sender and from fields are
+     *  * 'mailer' - Custom mailer class to use, if no namespace is spesified a class with same name is searched
```

Comment:
That's the `bolognaise` accent! 

Doc comment changed, should be more clear now .. not just for the typo